hw_5/hw_5.1.js

```diff
@@ -0,0 +1,40 @@
+/*
+Task 1.
+1. Создайте цикл, который выведет в консоль цифры от 10 до 0
+2. Написать скрипт, который выведет 5 строк в консоль таким образом, чтобы в первой строчке выводилось :), во второй :):) и так далее
+  Пример в консоли:
+  :)
+  :):)
+  :):):)
+  :):):):)
+  :):):):):)
+
+  Для Оксаны и ее друзей: также реализовать циклом while
+  Рекоммендация: попробуйте метод .repeat()
+*/
+
+// subtask 1
+let counter = 10;
+for (let i = counter; i >= 0; i--) {
+	console.log(i);
+}
+
+// subtask 2
+let rowAmount = 5;
+let pattern = ':)';
+let output = '';
+
+for (let i = 0; i < rowAmount; i++) {
```

Comment:
```suggestion
for (let i = 0; i < 5; i++) {
```
You can set condition without declaring variable / const

Reply:
But its ok to have this-like depending on a variable)

Reply:
It's okay, of course

Reply:
@VladimirLQA yeh, I know it) 
but it seems to me to be easier managed if there is a variable outside the code block.
I would like to leave the such code view, if you are OK with that)